pom.xml

```diff
@@ -100,8 +105,8 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Since we're using newer version of the plugin now, we probably should use `<maven.compiler.release>11</maven.compiler.release>` instead

Reply:
done